scores/Maria_durch_ein'_Dornwald_ging/Maria_durch_ein'_Dornwald_ging(Distler,_Hugo)/Makefile

```diff
@@ -0,0 +1,61 @@
+SHELL=/bin/bash
```

Comment:
Could this be a Symlink to a second makefile template for pieces including commentaries?